.travis.yml

```diff
@@ -7,19 +7,13 @@ addons:
   chrome: stable
 
 cache:
-  bundler: true
-  apt: true
   directories:
     - "$HOME/.composer/cache"
-    - "$HOME/.console"
     - "$HOME/.drush/cache"
-    - "$HOME/.nvm"
-    - "$TMPDIR/blt-sandbox-instance/.rules"
```

Comment:
We don't use Drupal Console, NVM, or long phpunit tests any more. The history on these are here:
- https://github.com/acquia/blt/commit/d48c277d7479c5e266d821d5010471d50d7e4994
- https://github.com/acquia/blt/commit/71cdb5ea7e987a313a0f208dabf52ca7b1a514c5
- https://github.com/acquia/blt/pull/2545